js_modules/dagster-ui/packages/ui-core/src/assets/AssetActionMenu.tsx

```diff
@@ -68,6 +78,13 @@ export const AssetActionMenu = memo((props: Props) => {
       {deletePartitions.element}
       <Popover
         position="bottom-right"
+        targetTagName="div"
+        targetProps={{
```

Comment:
This feels a little odd, is this to avoid specifying these on line 147? Could they also be removed from line 143?